gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/DistributedSQLTestBase.java

```diff
@@ -1656,15 +1656,15 @@ public int startNetworkServer(int vmNum, String serverGroups,
    */
   protected void startNetworkServer(int vmNum, String serverGroups,
       Properties extraProps, int netPort) throws Exception {
-    assertTrue("VM number should be >= 1", vmNum >= 1);
+    assertTrue("VM number should be >= 0", vmNum >= 0);
```

Comment:
Good to have a separate API with name startNetworkServerOnLocator (both may share common code). That would be less confusing for developers, going forward.

Reply:
Ok, will add.